### PR TITLE
added in the URR model

### PR DESCRIPTION
**Pull request by Christian-Vavuris**

### CODE REVIEW

File: models/UserRecipeRating.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            allowNull: false,
+            autoIncrement: true,
+            references: {
+                model: 'User',
+                key: 'user_id'
+            }
+        }, 
+        recipe_id: {
+            type: DataTypes.INTEGER,
+            autoIncrement: true,
```
Comment (vanessalane) [inline models/UserRecipeRating.js:25]: you can take this out!

File: models/UserRecipeRating.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        urr_id: {
+            type: DataTypes.INTEGER,
+            primaryKey: true,
+            allowNull: false,
+            autoIncrement: true
+        }, 
+        user_id: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+            autoIncrement: true,
```
Comment (vanessalane) [inline models/UserRecipeRating.js:17]: you can take this out!

File: models/UserRecipeRating.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            references: {
+                model: 'User',
+                key: 'user_id'
+            }
+        }, 
+        recipe_id: {
+            type: DataTypes.INTEGER,
+            autoIncrement: true, 
+            references: {
+                model: 'recipe',
```
Comment (vanessalane) [inline models/UserRecipeRating.js:27]: not sure if this will throw an error, but recipe might be capitalized

File: models/UserRecipeRating.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            type: DataTypes.DECIMAL,
+            allowNull: false
+        },
+    },
+    {
+        sequelize,
+        timestamps: false,
+        freezeTableName: true,
+        underscored: true,
+        modelName: 'Recipe'
```
Comment (vanessalane) [inline models/UserRecipeRating.js:41]: this should match the name of your model, so UserRecipeRating

Comment (Christian-Vavuris) [reply models/UserRecipeRating.js:41]: :gigantic_brain:

**Pull request merged**